app/views/static_pages/cookies.html.erb

```diff
@@ -67,21 +67,21 @@
       </thead>
       <tbody>
         <tr class="govuk-table__row">
-          <td class="govuk-table__cell">XXXX_ga</td>
+          <td class="govuk-table__cell">_ga</td>
           <td class="govuk-table__cell">This helps us count how many people visit <%= t("service_name") %> by tracking if you’ve visited before</td>
           <td class="govuk-table__cell">After 2 years</td>
         </tr>
         <tr>
           <tr class="govuk-table__row">
-          <td class="govuk-table__cell">XXXX_gid</td>
+          <td class="govuk-table__cell">_gid</td>
           <td class="govuk-table__cell">This helps us count how many people visit <%= t("service_name") %> by tracking if you’ve visited before</td>
           <td class="govuk-table__cell">After 24 hours</td>
```

Comment:
Are we confident that these are the cookies used by the tag manager, too?

Reply:
When we were looking at implementing Tag Manager it gave people too much opportunity to be naughty so we changed it to standard Google Analytics. It is using the Google Analytics Global Tag, not Google Tag Manager.

Reply:
Ah ok. It looks to me like we're still including the tag manager script in the header, though. Or am I mistaken?